tests/e2e/pageObjects/AccountNameEditPage.js

```diff
@@ -0,0 +1,43 @@
+const { format } = require("util");
+
+const config = require("../../cucumber.conf.js");
+
+class AccountNameEditPage {
+
+  constructor() {
+    this.baseUrl = config.tandoorURL;
+    this.settingURL = this.baseUrl + 'search/'
+    this.spaceOverviewURL = this.baseUrl + 'settings/';
+    this.navBarMenuSelector = '//i[contains(@class,"fa-user-alt")]//ancestor::a';
+    this.settingLinkSelector = "//a[@href='/settings/']";
```

Comment:
use quotes pattern consistently.
```suggestion
    this.navBarMenuSelector = '//i[contains(@class,"fa-user-alt")]//ancestor::a';
    this.settingLinkSelector = '//a[@href="/settings/"]';
```